src/engine/renderer/glsl_source/reliefMapping_fp.glsl

```diff
@@ -79,13 +88,16 @@ vec3 NormalInTangentSpace(sampler2D normalMap, vec2 texNormal)
 #endif
 
 	return normal;
+#else // !r_normalMapping
+	return vec3(1.0, 1.0, 1.0);
```

Comment:
I believe it should be (0, 0, 1), based on what you said the flat texture was. Presumably this means pointing directly outward. (1, 1, 1) is definitely not right as it is not a normalized vector (i.e., length 1).

Reply:
yes, the code was already there, I probably just forgot to push it.